jira/client.py

```diff
@@ -1750,6 +1792,15 @@ def createmeta(
             Dict[str, Any]
 
         """
+        if self._version >= (9, 0, 0):
+            raise JIRAError(f"Unsupported JIRA version: {self._version}")
+        elif self._version >= (8, 4, 0):
+            warnings.warn(
+                "This API have been deprecated in JIRA 8.4 and is removed in JIRA 9.0."
+                "Use 'createmeta_issuetypes' and 'createmeta_fieldtypes' instead.",
+                DeprecationWarning,
+            )
```

Comment:
we probably want to change the stack level here. Unfortunately the python default isn't super helpful. 
We can check this is a mocked test

Reply:
Sorry, I don't have a lot of experience with Python, so I don't understand what does _stack level_ in this context mean.
Should I do something like:
```python
 warn(
            "This function will fail from Jira 9+. "
            "Use issue_createmeta_issuetypes or issue_createmeta_fieldtypes instead.",
            DeprecationWarning,
            stacklevel=2,
        )
```
(taken from https://github.com/atlassian-api/atlassian-python-api/pull/1031/files)

Reply:
yep exactly right, here's a useful video explaining this situation: https://www.youtube.com/watch?v=CtFdXBEwYfk